example/helloworld.proto

```diff
@@ -8,10 +8,17 @@ service Greeter {
   rpc SayHello (HelloRequest) returns (HelloReply) {}
 }
 
+enum Color {
+  RED = 0;
+  BLUE = 1;
+  GREEN = 2;
+}
+
 message HelloRequest {
-  string name = 1;
+  string displayname = 1;
```

Comment:
should be lower camel case as `displayName`

Reply:
fixed with latest commit.